src/manager/components/CommentList/style.css

```diff
@@ -0,0 +1,10 @@
+.comment-content p {
```

Comment:
Could you try different Markdown elements and see they looks great here.
Try to match the style to look like GitHub if needed.